Last call post for grants/cfp, publish volunteer info (#2411)
writethedocs/www: docs/conf/berlin/2025/news/cfp-grants-closing.md

---
template: {{year}}/generic.html
og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
---

```{eval-rst}
.. post:: July 14, 2025
   :tags: {{shortcode}}-{{year}}
```

# Last call for Call for Proposals and Opportunity Grants

> We're excited to see all your proposals come in our Berlin conference!

Reviewer: ```suggestion
We're excited to see all the proposals you've sent in for our Berlin conference!
```
